KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/HighlightedText.java

```diff
@@ -359,16 +362,33 @@ public String getText() {
 
             highlightField = LuceneQuery.HIGHLIGHT_FIELD;
             if (isLiteral) {
-                //if the query is literal try to get solr to do the highlighting
-                final String highlightQuery = keywords.stream()
-                        .map(HighlightedText::constructEscapedSolrQuery)
-                        .collect(Collectors.joining(" "));
-
-                q.setQuery(highlightQuery);
-                q.addField(highlightField);
-                q.addFilterQuery(filterQuery);
-                q.addHighlightField(highlightField);
-                q.setHighlightFragsize(0); // don't fragment the highlight, works with original highlighter, or needs "single" list builder with FVH
+                if (2.2 <= indexSchemaVersion) {
+                    //if the query is literal try to get solr to do the highlighting
+                    final String highlightQuery = keywords.stream().map(s ->
+                        LanguageSpecificContentQueryHelper.expandQueryString(KeywordSearchUtil.escapeLuceneQuery(s)))
+                        .collect(Collectors.joining(" OR "));
+                    q.setQuery(highlightQuery);
+                    for (Server.Schema field : LanguageSpecificContentQueryHelper.getQueryFields()) {
+                        q.addField(field.toString());
+                        q.addHighlightField(field.toString());
+                    }
+                    q.addField(Server.Schema.LANGUAGE.toString());
+                    // in case of single term literal query there is only 1 term
+                    LanguageSpecificContentQueryHelper.configureTermfreqQuery(q, keywords.iterator().next());
+                    q.addFilterQuery(filterQuery);
```

Comment:
These 2 lines appear to be the same in both the "if" and "else" blocks. Perhaps they should be moved out of the if..else.